schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-2

```diff
@@ -0,0 +1,90 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for mobile contexts",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "mobile_context",
+		"format": "jsonschema",
+		"version": "1-0-2"
+	},
+	"type": "object",
+	"properties": {
+		"osType": {
+			"type": "string"
+		},
+		"osVersion": {
+			"type": "string"
+		},
+		"deviceManufacturer": {
+			"type": "string"
+		},
+		"deviceModel": {
+			"type": "string"
+		},
+		"carrier": {
+			"type": ["string", "null"]
+		},
+		"networkType": {
+			"type": ["string", "null"],
+			"enum": ["mobile", "wifi", "offline"]
+		},
+		"networkTechnology": {
+			"type": ["string", "null"]
+		},
+		"openIdfa": {
+			"type": ["string", "null"]
+		},
+		"appleIdfa": {
+			"type": ["string", "null"]
+		},
+		"appleIdfv": {
+			"type": ["string", "null"]
+		},
+		"androidIdfa": {
+			"type": ["string", "null"]
+		},
+		"physicalMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"description": "Total physical system memory in bytes"
+		},
+		"systemAvailableMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"description": "Available memory on the system in bytes (Android only)"
+		},
+		"appAvailableMemory": {
```

Comment:
I wonder if it would make sense to add a prefix in the name to state clearly that this field is platform specific. Something like `iOSAppAvailableMemory`. I don't think we have done anything like that before but it may help to avoid confusion when the data is extracted. Thoughts?

Reply:
That's a good point, I think it would be clearer. The counter argument is that maybe in the future we will be able to get the system available memory on iOS and app available memory on Android which would make it difficult to reuse these properties.